charts/kafka/README.md

```diff
@@ -0,0 +1,502 @@
+<!---
+Copyright 2020 Crown Copyright
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Originally copied from: https://github.com/helm/charts/blob/6c85be7b88748171afd17affe8b1b57c66bf66a2/incubator/kafka/README.md
+--->
+
+# Apache Kafka Helm Chart
+
+This chart was cloned from: https://github.com/helm/charts/tree/master/incubator/kafka
+I have also added a testclient to test operation of the Kafka cluster.
+This particular Helm chart has a dependency on zookeeper. This dependency was resolved
+by running:
+```bash
+helm dependency update
+```
+This pulled in zookeeper-2.1.0.tgz 
+This has been unzipped to ./charts.zookeeper to prevent the need for performing
+a dependency update.
+
+After deployment of the cluster you should see the following:
+
+```bash
+kubectl get all --namespace=dev
+NAME                                    READY   STATUS    RESTARTS   AGE
+pod/palisade-kafka-0                    1/1     Running   1          30m
+pod/palisade-kafka-1                    1/1     Running   0          28m
+pod/palisade-kafka-2                    1/1     Running   0          27m
+pod/palisade-kafka-testclient           1/1     Running   0          30m
```

Comment:
Can we make it so this test client is only deployed when you specify some flag so we only deploy in dev/test environments and not in prod